src/main/kotlin/Calculator.kt

```diff
@@ -0,0 +1,48 @@
+import java.math.BigDecimal
+import java.math.MathContext
+import java.util.Stack
+
+class Calculator {
+    private fun add(i: BigDecimal, j: BigDecimal): BigDecimal =
+        i.add(j)
+
+    private fun subtract(i: BigDecimal, j: BigDecimal): BigDecimal =
+        i.subtract(j)
+
+    private fun multiply(i: BigDecimal, j: BigDecimal): BigDecimal =
+        i.multiply(j)
+
+    private fun divide(i: BigDecimal, j: BigDecimal): BigDecimal =
+        i.divide(j, MathContext.DECIMAL64)
+
+    fun calculate(strNum1: String?, strNum2: String?, operator: String): String {
+        val i = strNum1?.toBigDecimalOrNull() ?: throw IllegalArgumentException()
+        val j = strNum2?.toBigDecimalOrNull() ?: throw IllegalArgumentException()
+        return when (operator) {
+            Operator.ADD.symbol -> add(i, j)
+            Operator.MINUS.symbol -> subtract(i, j)
+            Operator.MULTIPLE.symbol -> multiply(i, j)
+            Operator.DIVIDE.symbol -> divide(i, j)
+            else -> throw IllegalArgumentException()
+        }.toString()
+    }
+
+    fun calculateStringInput(input: String): String {
+        val stack = Stack<String>()
+        val number = Stack<String>()
+        input.split(" ").map { s ->
+            if (Operator.getOperators().contains(s)) {
```

Comment:
```suggestion
            if (Operator.isContains(s)) {
```
`getOperators()` 를 이용해 operator 값을 `Calculator` 클래스로 가져오기보다는 `Operator` 클래스에게 메시지를 던저 결과만 받아오도록 하는건 어떨까요?